helix-term/src/commands/expansions.rs

```diff
@@ -0,0 +1,196 @@
+use std::borrow::Cow;
+
+use anyhow::Result;
+use helix_core::coords_at_pos;
+use helix_view::Editor;
+
+pub fn expand_args<'a>(
```

Comment:
`expand_args` might lead to some confusion about what it does. My previous suggestion of an `variables.rs` just and `expand` should read `variables::expand(...)`, which to me is more clear what its doing, acting on only variables, not all `args` which it could suggest.

Reply:
done :)